app/assets/stylesheets/semantic-ui/globals/_variables.scss

```diff
@@ -1,4 +1,4 @@
 $import-google-fonts: true !default;
-$font-url: 'https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic&subset=latin,latin-ext' !default;
-$font-name: 'Lato' !default;
-$font-family: $font-name, 'Helvetica Neue', Arial, Helvetica, sans-serif !default;
+$font-url: 'proxima-nova' !default;
```

Comment:
This won't work unless you make sure that the font is loaded elsewhere. I don't think everyone watching the docs will have the font installed locally.